Web/Dockerfile

```diff
@@ -23,4 +23,5 @@ WORKDIR /usr/share/nginx/html
 RUN rm -rf ./*
 
 COPY --from=baseweb /home/node/app/build .
+EXPOSE 80
```

Comment:
how come we had to add this?

Reply:
Since we are using Nginx. It worked before since I was sending the cmd yarn start. It didn't actually use the nginx site thats hosted in the dockerfile lmfao checkout docker-compose line 28. I have a feeling thats why I was having trouble deploying new changes. Now we are compiling the production build for react and then having the docker-compose expose the nginx site on port 80.

Reply:
Server is still using the yarn start command but we should probably implement production builds to minimize the size and so we aren't dependent on the gitrepo. I think we can be at a point where our production server doesn't clone the repo but rather gets the 'prod' image and creates the docker containers